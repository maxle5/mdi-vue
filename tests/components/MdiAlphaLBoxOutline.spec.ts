
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiAlphaLBoxOutline from "../../src/components/MdiAlphaLBoxOutline.vue";

test("MdiAlphaLBoxOutline snapshot", () => {
  const wrapper = mount(MdiAlphaLBoxOutline, {});
  expect(wrapper.html()).toMatchSnapshot();
});
