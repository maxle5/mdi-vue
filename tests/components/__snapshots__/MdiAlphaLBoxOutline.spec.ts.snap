// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiAlphaLBoxOutline snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M9,7H11V15H15V17H9V7M5,3H19A2,2 0 0,1 21,5V19A2,2 0 0,1 19,21H5A2,2 0 0,1 3,19V5A2,2 0 0,1 5,3M5,5V19H19V5H5Z" style="fill: currentColor;"></path>
</svg>"
`;
